.rubocop_todo.yml

```diff
@@ -11,6 +11,18 @@
 Metrics/ClassLength:
   Max: 184
 
-# Offense count: 8
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: AutoCorrect, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 91424
```

Comment:
yeah shall we ignore this? move it to config rather than todo?

Reply:
Yeah, I think it makes sense to ignore it altogether. That ok? Might have to move the factory_bot config because we don't want that ignored.

Reply:
yeah, makes sense 👍

Reply:
:nods: added

Reply:
thanks!